### ISSUE TITLE
New Example: Use Mongoose Schema

### ISSUE THREAD

**Issue by mikeerickson** [formly-js/angular-formly-website#12]
I was doing some comparison between 'schema form' and 'formly' and it dawned on me, it would be nice to use an existing Mongoose schema as the source for fields, validation, etc.

**Comment by kentcdodds** [comment]
Totally, terrific example idea. Would probably be very similar to http://angular-formly.com/#/example/very-advanced/oim

**Comment by mikeerickson** [comment]
You are indeed correct, that is basically what needs to be done (using the OIM approach).   I will mess around with this and see what I can come up with.

FWIW, this is a 'comparison' issue with the other tool as it is based on schema (it doesn't have Mongoose connection either so ... )

**Comment by kentcdodds** [comment]
I should note, that there's an entire repo already initialized to add json-schema support to angular-formly as a plugin: https://github.com/formly-js/angular-formly-json-schema

**Comment by mikeerickson** [comment]
Fantastic, that looks yummy!

**Comment by mikeerickson** [comment]
Can you please confirm contents of this repo?   There are only two files (README and LICENSE) ??

**Comment by kentcdodds** [comment]
Yes, this is true. I started a branch a while ago in angular-formly core to add support but didn't have time to work on it. Obviously it'll be implemented a bit differently now, but here's the starting point: https://github.com/formly-js/angular-formly/commit/14db7819173cb7fa133a59e7f5668407adb6e0f1

There may be changes necessary to angular-formly to enable this, and that's fine. But I'd like it to be fairly minimal.

**Comment by mikeerickson** [comment]
Ahhhhh, misunderstood what you said.

**Comment by kentcdodds** [comment]
Nah, I still want an example of doing something like this with mongoose :-)

**Comment by mikeerickson** [comment]
Well, this I will do just that :)
